docs/creating_your_tooltip.md

```diff
@@ -85,7 +88,7 @@ You can create your tooltip using [`vegaTooltip`](APIs.md#tooltip). This functio
 vegaEmbed("#vis", spec, opt)
   .then(function(result) {
     // result.view is the Vega View
-    vegaTooltip(result.view);
+    vegaTooltip.default(result.view);
```

Comment:
Default is only necessary if you don't use a bundler as I explain below. I'm not sure what the cleanest way to explain this is.

Reply:
ah, ic - I think this note wasn't there yesterday... feel free to drop this PR then. 
Perhaps there is a way to highlight this note in the markdown? spent some time figuring that out

Reply:
Thanks for the PR! I think this is a bit confusing. You can highlight in markdown for example with `**`.